.github/workflows/pr_tests.yml

```diff
@@ -113,7 +113,7 @@ jobs:
     - name: Run example PyTorch CPU tests
       if: ${{ matrix.config.framework == 'pytorch_examples' }}
       run: |
-        python -m pip install peft
+        python -m pip install peft==0.7.1
```

Comment:
Check https://huggingface.slack.com/archives/C04L3MWLE6B/p1706601893641009

Reply:
Could it make sense to do this is a seperate PR?

Reply:
Yeah. Will be removed altogether.